cmd/dex/serve.go

```diff
@@ -154,6 +154,13 @@ func runServe(options serveOptions) error {
 		return fmt.Errorf("failed to register gRPC server metrics: %v", err)
 	}
 
+	var defaultConnectorByClientID map[string]string
+	if filePath := os.Getenv(defaultConnectorsFileEnv); filePath != "" {
```

Comment:
What's the reasoning behind putting this in it's own config file vs being added to the existing config file? Fragmentation seems more awkward, usability wise.